cypress/fixtures/fireboltCalls/discovery.js

```diff
@@ -72,3 +73,864 @@ exports.DISCOVERY_CLEAR_CONTENTACCESS = {
+          {
+            type: 'program-lineup',
+            id: 'partner.com/availability/123',
+            catalogId: 'app1',
+            startTime: '2021-04-23T18:25:43.511Z',
+            endTime: false,
+          },
+        ],
+      },
+    },
+    CONTENTACCESS_ENTITLEMENTS_INVALID_ENTITLEMENTID_TRUE: {
+      ids: {
+        entitlements: [
+          {
+            entitlementId: true,
+            startTime: '2025-01-01T00:00:00.000Z',
+            endTime: '2025-01-01T00:00:00.000Z',
+          },
+        ],
+      },
+    },
+    CONTENTACCESS_ENTITLEMENTS_INVALID_ENTITLEMENTID_INTEGER: {
+      ids: {
+        entitlements: [
+          {
+            entitlementId: 123,
+            startTime: '2025-01-01T00:00:00.000Z',
+            endTime: '2025-01-01T00:00:00.000Z',
+          },
+        ],
+      },
+    },
+    CONTENTACCESS_ENTITLEMENTS_INVALID_START_TIME: {
+      ids: {
+        entitlements: [
+          {
+            entitlementId: '123',
+            startTime: '01022023',
+          },
+        ],
+      },
+    },
+    CONTENTACCESS_ENTITLEMENTS_INVALID_END_TIME: {
+      ids: {
+        entitlements: [
+          {
+            entitlementId: '123',
+            endTime: '01022023',
+          },
+        ],
+      },
+    },
+  },
+};
+exports.DISCOVERY_ONSIGNIN = {
+  method: 'manage_discovery.onSignIn',
+  params: {},
+};
+
+exports.NOTIFY_USER_HAS_SIGNED_IN_WITH_ENTITLEMENTS = {
+  method: 'discovery.signIn',
+  params: this.DISCOVERY_COMMON_VARIABLES.DISCOVERY.SIGNIN_WITH_ENTITILEMENTS,
+};
+
+exports.NOTIFY_USER_HAS_SIGNED_IN_WITHOUT_ENTITLEMENTS = {
+  method: 'discovery.signIn',
+  params: {},
+};
+
+exports.DISCOVERY_ONSIGNOUT = {
+  method: 'manage_discovery.onSignOut',
+  params: {},
+};
+
+exports.NOTIFY_USER_HAS_SIGNED_OUT = {
+  method: 'discovery.signOut',
+  params: {},
+};
+
+exports.SIGNIN_FOR_DISCOVERY = {
+  method: 'discovery.signIn',
+  validationJsonPath: 'result',
+  content: true,
+};
+
+exports.SIGNOUT_FOR_DISCOVERY = {
+  method: 'discovery.signOut',
+  validationJsonPath: 'result',
+  content: true,
+};
+
+exports.NOTIFY_WATCHED_CONTENT_WITH_ONLY_ENTITYID = {
+  method: 'discovery.watched',
+  params: this.DISCOVERY_COMMON_VARIABLES.DISCOVERY.WATCHED_ONLY_ENTITY_ID,
+};
+
+exports.NOTIFY_WATCHED_CONTENT_WITH_ENTITYID_PROGRESS = {
+  method: 'discovery.watched',
+  params: this.DISCOVERY_COMMON_VARIABLES.DISCOVERY.WATCHED_ENTITY_ID_PROGRESS,
+};
+
+exports.NOTIFY_WATCHED_CONTENT_WITH_ENTITYID_PROGRESS_STATUS = {
+  method: 'discovery.watched',
+  params: this.DISCOVERY_COMMON_VARIABLES.DISCOVERY.WATCHED_ENTITY_ID_PROGRESS_STATUS,
+};
+
+exports.NOTIFY_WATCHED_CONTENT_WITH_ENTITYID_PROGRESS_STATUS_DATE = {
+  method: 'discovery.watched',
+  params: this.DISCOVERY_COMMON_VARIABLES.DISCOVERY.WATCHED_ENTITY_ID_PROGRESS_STATUS_DATE,
+};
+
+exports.TRUE_FOR_WATCHED_CONTENT_IN_DISCOVERY = {
+  method: 'discovery.watched',
+  validationJsonPath: 'result',
+  content: true,
+};
```

Comment:
Shouldnt this be moved to a constant and placed in some js file ? something like defaults.js ?

Reply:
yes, we can store these values in a _defaults.js_ file and import it into the required _<moduleName>.js_ file to use them. Similar approach to _errorContentObjects.j_s for managing invalid type parameters.